scripts/notify.lic

```diff
@@ -30,7 +30,7 @@
     # because it is in a different thread
     Notify::Sounds.set "zelda/email"
   
-    Notify.notify(from: "taters", body: "i'm important", type: Script.current.name)
+    Notify.notify("Ondreian", from: "taters", body: "i'm important", type: Script.current.name)
```

Comment:
I'm not really liking this API change so far. First of all, it is a breaking so all of my existing uses are borked. This is unfortunate, but not a deal breaker. Secondly, I have yet to want to notify from a character other than the one making the call. Is there any way we could convert this into an optional field and fallback to `Char.name` to preserve the existing API?

Reply:
The problem is, it's used by the tcp sink also, but i agree, a key argument makes more sense here that defaults to the current character makes more sense, i will fix